README.md

```diff
@@ -184,6 +184,14 @@ It is highly recommended you set the following environment values before startin
 | AUTO_REBOOT_CRON_EXPRESSION  | Setting affects frequency of automatic updates. | 0 0 \* \* \* | Needs a Cron-Expression - See [Configuring Automatic Backups with Cron](#configuring-automatic-reboots-with-cron) |
 | AUTO_REBOOT_ENABLED | Enables automatic reboots | false | true/false |
 | AUTO_REBOOT_WARN_MINUTES | How long to wait to reboot the server, after the player were informed. | 5 | !0 |
+| DISCORD_WEBHOOK_ID | Discord webhook id found after creating a webhook on a discord server ex: discord.com/api/webhooks/<webhook_id> | 0123456789012345678/xxxxxxxxxxxxxxxxxxxxxx | |
```

Comment:
I would change this to be the entire URL `https://discord.com/api/webhooks/xxxxxxxxxxxxxxxxxxxx/abcdef`. Makes it easier for users.

Also there is no default value.

Reply:
Thats an ez change, when I put the url in the readme tho I get a linting error for using a bare url. Anyway to get around that or we should ignore it?

About the default value, there should not be one as the value is unique and must be generated.

Reply:
Right there should be a default value. It looked to me like there was a default value listed in the Readme.